### PR TITLE
Feature/mobile

### PR DESCRIPTION
**Pull request by hansss**
- Mobile menu bar working. 
- No more overlap of divs on home page.
- Correct sizing for "Review questions" page

### CODE REVIEW

File: .idea/runConfigurations/Development__askup.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (elliottyates) [inline .idea/runConfigurations/Development__askup.xml]: The .idea files crept back into this feature branch! :) Review our conversation from March 16, highlighted here:
https://askup.slack.com/messages/development/search/filter%20out%20the%20.idea/%20folder%20by%20default/

Then remove those from the feature branch (you can simply delete the .idea directory and then commit those deletions).

File: app/assets/stylesheets/home.css.scss [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+  .question-cell{
+      padding-left: 0;
+      padding-right: 0;
+      width: 100%;
+      float: left;
+      margin-left: 0% !important;
+  }
 
-  h1 {
+   h1 {
```
Comment (elliottyates) [inline app/assets/stylesheets/home.css.scss]: Seems like this has some unnecessary indentation?

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -2,29 +2,46 @@
...
 
+    <div class="row jumbotron one-edge-shadow">
+        <div class="container-row">
+          <div class="cell">
+            <h1>Ask<span>Up</span></h1>
+            <h3>Learn better, together.</h3>
+          </div>
+          <div class="question-cell">
+              <div class="center"><%= link_to "Generate Question", new_question_path, class: "btn btn-theme btn-lg" %></div>
+              <!--<div class="center">-->
```
Comment (elliottyates) [inline app/views/home/index.html.erb]: If you've tested the app and you know it's working, it's a good idea to remove all these comments for lines / elements we no longer need.

File: app/assets/stylesheets/home.css.scss [hunk 4]
```diff
@@ -1,24 +1,59 @@
...
+//  }
+
+
+  .container-row{
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 0;
+    margin-bottom: 0;
+    max-width: 62.5rem;
```
Comment (elliottyates) [inline app/assets/stylesheets/home.css.scss:25]: Using `rem` here, but for font and margin settings (below) using `em`. Why the difference?

File: app/assets/stylesheets/home.css.scss [hunk 5]
```diff
@@ -29,8 +64,9 @@
   }
 }
 
+
 #intro {
-  margin-top: 10vw;
+  margin-top: 3em;
   margin-bottom: 10vw;
```
Comment (roderickmorales) [inline app/assets/stylesheets/home.css.scss:70]: Viewport units are very very young. If you want the best support across browsers stick to: em for vertical spacing and % for horizontal spacing.

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -2,57 +2,47 @@
...
         </div>
         <div class="col-md-4">
           <i class="fa fa-university"></i>
           <h3>Why does it work?</h3>
           <p>Self-testing has been shown to be one of the most effective learning strategies. Generating questions will require you to recall information, identify the main points, reflect on how it relates to things you’ve previously learned, and generate your own answer—powerful techniques for learning!</p>
-          <!-- <p><br/><a href="#" class="btn btn-theme">More Info</a></p> -->
         </div>
       </div>
-    </div><!--/container -->
-   </div><!--/service -->
```
Comment (elliottyates) [inline app/views/home/index.html.erb:49]: Are you missing a `</div>` here?

**Pull request merged**